keycloak-controller/src/main/java/io/enmasse/keycloak/controller/KeycloakParams.java

```diff
@@ -8,41 +8,33 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.ByteArrayInputStream;
-import java.io.File;
-import java.io.IOException;
-import java.io.UncheckedIOException;
-import java.nio.file.Files;
 import java.security.KeyStore;
 import java.security.cert.CertificateFactory;
 import java.util.Map;
-import java.util.Optional;
 
 public class KeycloakParams {
 
     private static final Logger log = LoggerFactory.getLogger(KeycloakParams.class);
 
-    private final String host;
-    private final int httpPort;
+    private final String keycloakUri;
```

Comment:
Again, not a blocker, but it seems like the same piece of data is called different things in various places, httpUrl, keycloakUri, oauthUrl... might be clearer to just use the same throughout?